ably/http/http.py

```diff
@@ -128,14 +183,18 @@ def __init__(self, ably, options):
         options = options or {}
         self.__ably = ably
         self.__options = options
-
         self.__session = requests.Session()
         self.__auth = None
 
-    @fallback
-    @reauth_if_expired
+    #@fallback
+    #@reauth_if_expired
```

Comment:
More of a question than anything else, but what did @fallback previously do, and why is now commented out as opposed to removed?